src/internal/translations/pt-BR.json

```diff
@@ -79,7 +83,7 @@
     "properties.yaxis.title": "Eixo Y",
     "properties.axis.labels": "rótulos",
     "properties.axis.rotation": "Rotação",
-    "properties.axis.name": "nome",
+    "properties.axis.name": "nme",
```

Comment:
Probably an error – the pt-PT does not have this change.

Reply:
Same error again. Fixed